Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -72,6 +72,15 @@
   description: A classic-style weapon infused with advanced atmos technology to allow it to set targets on fire.
   price: 10
 
+- type: uplinkListing
+  id: UplinkSyndieLaserCarbine
+  category: Weapons
+  itemId: WeaponSyndieLaserCarbine
+  listingName: Syndicate Laser Gun
+  description:  A stolen laser gun equipped with several state-of-the-art modifications that allows it to recharge itself slowly. 
+  price: 12
```

Comment:
This price feels a bit steep compared to the other weapons. It can probably cost less than this.

Reply:
Got any suggestions? I'm thinking 10, but i'm honestly pretty unsure.